kafka-client/src/test/java/dev/responsive/kafka/integration/RowLevelTtlIntegrationTest.java

```diff
@@ -202,35 +193,13 @@ public void shouldApplyRowLevelTtlForKeyAndValue() throws Exception {
   }
 
   private ResponsiveKafkaStreams buildStreams(final Map<String, Object> properties) {
-    final StreamsBuilder builder = new StreamsBuilder();
-
-    final KStream<String, String> input = builder.stream(inputTopic());
-    input.process(new TtlProcessorSupplier(), STORE_NAME)
-        .to(outputTopic());
-
-    return new ResponsiveKafkaStreams(builder.build(), properties);
-  }
-
-  @SuppressWarnings("checkstyle:linelength")
-  private static class TtlProcessorSupplier implements ProcessorSupplier<String, String, String, String> {
-
-    @Override
-    public Processor<String, String, String, String> get() {
-      return new TtlDeduplicator();
-    }
+        final var params = ResponsiveKeyValueParams.fact(STORE_NAME).withTtlProvider(
```

Comment:
no actual changes to this test at all, just moving some things to the testutils and getting rid of  the duplicate code here